packages/compass-web/src/entrypoint.tsx

```diff
@@ -51,16 +53,10 @@ import {
 } from '@mongodb-js/atlas-service/provider';
 import type { AtlasUserInfo } from '@mongodb-js/atlas-service/provider';
 import { AtlasAiServiceProvider } from '@mongodb-js/compass-generative-ai/provider';
-import type {
-  ConnectionStorage,
-  ConnectionInfo,
-} from '@mongodb-js/connection-storage/provider';
-import {
-  ConnectionStorageProvider,
-  ConnectionInfoProvider,
-} from '@mongodb-js/connection-storage/provider';
+import { ConnectionStorageProvider } from '@mongodb-js/connection-storage/provider';
 import { useLoggerAndTelemetry } from '@mongodb-js/compass-logging/provider';
 import CompassConnections from '@mongodb-js/compass-connections';
+import { CompassWebConnectionStorage } from '@mongodb-js/connection-storage/renderer';
```

Comment:
We souldn't be importing stuff from the renderer in the web, this is an electron specific export, for now I suggest we just keep the compass web implementation in the same file and see if we want to change that at some point

Reply:
Moved this to compass-web